www/admin/site-edit.php

```diff
@@ -211,10 +211,11 @@
 $themelist = [];
 $themes    = scandir(nZEDb_WWW . "/themes");
 foreach ($themes as $theme) {
-	if (strpos($theme, ".") === false && is_dir(nZEDb_WWW . "/themes/" . $theme)) {
+	if (strpos($theme, ".") === false && is_dir(nZEDb_THEMES . $theme) && ucfirst($theme) === $theme) {
```

Comment:
`ucfirst($theme) === $theme`

This should be reverted, since this can cause issues with people using custom themes. It's undocumented and nothing is displayed to hint people with custom themes why their theme is not loading.